pkgs/by-name/re/redfishtool/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchPypi
+, python3
+}:
+
+let
+  pname = "redfishtool";
+  version = "1.1.8";
+in
+python3.pkgs.buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-X/G6osOHCBidKZG/Y2nmHadifDacJhjBIc7WYrUCPn8=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [ requests python-dateutil ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A Python34 program that implements a command line tool for accessing the Redfish API.";
+    homepage = "https://github.com/DMTF/Redfishtool";
+    license = licenses.unfree;
```

Comment:
Skimming the upstream repo, it seems like it is supposed to be BSD 3-clause, which agrees with what the [Debian](https://metadata.ftp-master.debian.org/changelogs/main/r/redfishtool/redfishtool_1.1.0-3_copyright) folks have on their repositories. Is this actually unfree?

Reply:
i was unsure when I read their license so I based the license on the Package Request which stated that it was proprietary. Will resolve all these issues. Thanks